src/app/components/Card/Card.tsx

```diff
@@ -0,0 +1,99 @@
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Paper, { PaperProps } from '@mui/material/Paper'
+import Divider from '@mui/material/Divider'
+import useTheme from '@mui/material/styles/useTheme'
+import Typography from '@mui/material/Typography'
+import { darken, lighten } from '@mui/material/styles'
+
+import { ICard, ICrop, IPlayedCrop } from '../../../game/types'
+import { isCrop } from '../../../game/types/guards'
+import { cards, isCardImageKey, ui } from '../../img'
+
+import { CardCropText } from './CardCropText'
+
+export type CardProps = PaperProps & {
+  card: ICard
+  size?: number
```

Comment:
typescript question here... what does it mean to have the `&` when both have a "card" attribute with a different type? feels like "&" is acting like an "or"..?